libraries/ESP8266WebServer/src/ESP8266WebServer-impl.h

```diff
@@ -45,7 +44,8 @@ static const char Content_Length[] PROGMEM = "Content-Length";
 
 template <typename ServerType>
 ESP8266WebServerTemplate<ServerType>::ESP8266WebServerTemplate(IPAddress addr, int port)
-: _server(addr, port)
+: _corsEnabled(false)
```

Comment:
`_corsEnabled` needs to be last in the initialization list because you put it last in members (CI is not happy).

Also, for clarity, can you replace `_ec` by `enable` ?

Reply:
yes you're right ! i will correct this... it was my first pull :-) how to correct ? will I post a new pull request ?

Reply:
Just commit and push to the same branch and it will automatically pick it up here.

Reply:
ok Done